pkg/eventemitter/cloudevent_http_handler.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2022 The KEDA Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventemitter
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/protocol"
+	"github.com/go-logr/logr"
+)
+
+type CloudEventHTTPHandler struct {
```

Comment:
Could you please add here a short describtion what is the struct about?

Also adding a general comment at the beginning would be nice, see this for an example: https://github.com/kedacore/keda/blob/fc3f31c3d3c74351eb1da5051859368b422890af/pkg/scaling/modifiers/formula.go#L17